OpenRA.Mods.Common/Scripting/Properties/TransportProperties.cs

```diff
@@ -42,9 +42,9 @@ public TransportProperties(ScriptContext context, Actor self)
 
 		[ScriptActorPropertyActivity]
 		[Desc("Command transport to unload passengers.")]
-		public void UnloadPassengers()
+		public void UnloadPassengers(Target? destination)
```

Comment:
Setting a default null value should be enough to fix the API breakage.

Reply:
OK, that's a silly mistake. The whole point of making `destination` nullable was to avoid breaking the API, but then I forgot to actually add a default. :man_facepalming: